docs/data-import.rst

```diff
@@ -36,7 +36,7 @@ HistomicsML datasets should be stored in subdirectories within a single master d
   |----- classifiers/
       |----- tmp/
```

Comment:
@slee172  - what is the /tmp directory?

Reply:
Stores files generated by reporting tools for download - remove from documentation.